include/uxr/agent/types/XRCETypes.hpp

```diff
@@ -10315,6 +10372,7 @@ class HEARTBEAT_Payload
 private:
     uint16_t m_first_unacked_seq_nr;
     uint16_t m_last_unacked_seq_nr;
+    uint8_t m_stream_id;
```

Comment:
With the new approach that we are following with the submessage (about sending the alignment after the submessage). We will send 8 of payload instead of 5. Is this overhead contemplated?. By other side, if the Heartbeats and Acknacks are always send in separated submessages, this offset after the message can be ommited and only 5 bytes can be sent.

Reply:
The Agent is able to received these submessage with the offset. Obviously, `HEARTBEATs` and `ACKNACKs` are always sent in separated submessages.